manufacturing/enclosed_build/display_panel/README.md

```diff
@@ -1,18 +1,39 @@
-(6.35 mm) was more than minimum hole depth specification (5.72 mm).
-Hence the corresponding outer diameter for insertion i.e. acrylic mounting diameter would be 5.94 mm, which is
-reflected in the CAD.
+This readme contains the details, BOM, assembly instructions for one display panel assembly for the enclosed build. 
+
+## Design Rationale
+
+The display panel accommodates the LCD touchscreen display and the oxygen inlet port. The display panel assembly should provide an easily accessible way to the primary user interface - the touchscreen display. It accomplishes this by mounting the display assembly in a custom cut transparent acrylic sheet which also allows one to view the internals.  
+### Display Panel mounting choice 
+It was decided that heat set inserts be used to help screw the display panel to the sheet metal enclosure tray. This was done since the holes in the acrylic lacked any internal threads. Tapped/plastic threads are easy to overload and strip out. Metal threads provide a reliable way of fastening plastic and acrylic parts. The heat set inserts will resist pull-out and torque-out loads due to knurled external surface features designed to interface with the bulk acrylic. Also, using screws and nuts wasn't very feasible due to the lack of space between the mounted display (i.e. when attached to the sheet metal equipment tray) and the front face of the sheet metal enclosure. The sizing for the mounting holes was based on [this](https://www.pemnet.com/fastening_products/pdf/sidata.pdf) catalog for heat set insert. M4 screws were chosen to mount the acrylic face plate to the sheet metal enclosure. For corresponding M4 nuts (refer page 6, metric table, row 6), length code 1 was chosen since acrylic thickness (6.35 mm) was more than minimum hole depth specification (5.72 mm). Hence the corresponding outer diameter for insertion i.e. acrylic mounting diameter would be 5.94 mm, which is reflected in the CAD.
+### LCD Display mounting choice
+The heat set inserts were not deemed necessary for the holes used to mount the LCD display to the acrylic plate due to the LCD's lighter weight. M2 screws, washers and locknuts were used; details below. Standard clearance hole for an M2 screw is 2.4mm which is reflected in the CAD.
+### Oxygen Inlet choice
+This ventilator relies on an external source of oxygen. The inlet port consists of a brass threaded check valve which allows only unidirectional flow into the ventilator. A barbed tubing adapter connects the internal pneumatic assembly to the oxygen source. The check valve and tubing adapter are connected through the acrylic plate via a through-wall straight connector that consists of a washer and nut in addition to the connector itself. Brass fittings were chosen since they offer superb corrosion resistance, a desirable trait in a high oxygen content environment, in addition to ability to withstand high pressures.
```

Comment:
This is also quite bloated. Do we care that it is "brass threaded"? "Check valve ensures one-way flow of oxygen."
That a "tubing adapter connects" is also obvious. The sentence after that is also not very inetresting. "Brass is non-corrosive and can withstand ...".